klaas/scripts/train_separation_model.py

```diff
@@ -50,15 +50,17 @@ def main(configuration_path, signal_path, background_path, predictions_path, mod
     n_cross_validations = config.get('n_cross_validations', 10)
 
     training_variables = config['training_variables']
-    
+
+    true_energy = config['true_energy']
```

Comment:
Better use `config.get('true_energy', 'MCorsikaEvtHeader.fTotalEnergy')` to make the key optional.